src/main/java/ui/issuepanel/PanelControl.java

```diff
@@ -39,6 +45,8 @@ public PanelControl(UI ui, Preferences prefs) {
                 setCurrentlySelectedPanel(Optional.of(e.panelIndex)));
         ui.registerEvent((PanelClickedEventHandler) e ->
                 setCurrentlySelectedPanel(Optional.of(e.panelIndex)));
+        ui.registerEvent((ShowRenamePanelEventHandler) e -> Platform.runLater(() ->
+                ((FilterPanel) getPanel(e.panelId)).showRenameTextField()));
```

Comment:
In that case, `PlatformEx.runAndWait` should be used at those places. We should try to keep application code agnostic to what is done in tests.

```java
// Use GlobalConfigsTest.java
PlatformEx.runAndWait(() -> UI.events.triggerEvent(new ShowRenamePanelEvent(0)));
type("Renamed panel");
```